Doppler.Sap/Mappers/BusinessPartner/BusinessPartnerForArMapper.cs

```diff
@@ -121,8 +93,8 @@ public SapBusinessPartner MapDopplerUserToSapBusinessPartner(DopplerUserDto dopp
                             ZipCode = dopplerUser.BillingZip != null ? dopplerUser.BillingZip.ToUpper() : "",
                             City = dopplerUser.BillingCity != null ? dopplerUser.BillingCity.ToUpper() : "",
                             Country = dopplerUser.BillingCountryCode != null ? dopplerUser.BillingCountryCode.ToUpper() : "",
-                            State = dopplerUser.GroupCode == 115 ? dopplerUser.BillingStateId.ToString() :
-                                    (dopplerUser.BillingStateId.HasValue ? (states.TryGetValue(dopplerUser.BillingStateId.Value, out var sapBillStateId) ? sapBillStateId : "99") : "99"),
+                            State = dopplerUser.GroupCode == 115 ? dopplerUser.BillingStateId :
+                                    (!string.IsNullOrEmpty(dopplerUser.BillingStateId) ? dopplerUser.BillingStateId : "99"),
```

Comment:
Do you need to check for the group code now?
```suggestion
                            State = !string.IsNullOrEmpty(dopplerUser.BillingStateId) ? dopplerUser.BillingStateId : "99",
```

Reply:
Just I am changing it. Because now we don't need to check it.it. Because now we don't check it.

Reply:
Done